src/server/acl.json

```
@@ -11,8 +11,128 @@
     "resources": [
       {
         "type": "rest",
-        "resourceId": "^/.*",
+        "resourceId": "^/$",
         "actions": ["create", "view", "edit", "delete"]
+      },
+      {
+        "type": "rest",
+        "resourceId": "^/api/credentials.*",
+        "actions": ["create", "view", "edit", "delete"]
+      },
+      {
+        "type": "rest",
+        "resourceId": "^/api/deliveryconfigs.*",
+        "actions": ["create", "view", "edit", "delete"]
+      },
+      {
+        "type": "rest",
+        "resourceId": "^/api/deliveryconfigtemplates.*",
+        "actions": ["create", "view", "edit", "delete"]
+      },
+      {
+        "type": "rest",
+        "resourceId": "^/api/deliverynodes.*",
+        "actions": ["create", "view", "edit", "delete"]
+      },
+      {
+        "type": "rest",
+        "resourceId": "^/api/routes.*",
+        "actions": ["create", "view", "edit", "delete"]
+      },
+      {
+        "type": "rest",
+        "resourceId": "^/api/routeaccesslists.*",
+        "actions": ["create", "view", "edit", "delete"]
+      },
+      {
+        "type": "rest",
+        "resourceId": "^/api/webmethods/ocsftp[/]?$",
+        "actions": ["create", "view", "edit", "delete"]
+      },
+      {
+        "type": "rest",
+        "resourceId": "^/api/webmethods/mappings/fromcanonical/[^/]+[/]?$",
+        "actions": ["create", "view", "edit", "delete"]
+      },
+      {
+        "type": "rest",
+        "resourceId": "^/api/webmethods/mappings/combinationmaps|batchservices|packagingservices|serializationservices|targetvalidation[/]?$",
```

Try to combine that and the two above into single endpoint:
```suggestion
        "resourceId": "^/api/webmethods/.*$",
```